oqs-template/generate.yml

```diff
@@ -417,7 +417,21 @@ sigs:
     #                'pretty_name': 'RSA3072',
     #                'oid': '1.3.9999.1.3',
     #                'code_point': '0xfe02'}]
+    #    composite:[{'name': 'p256',
+    #                'pretty_name': 'ECDSA p256',
+    #                'security': '128',
+    #                'oid': '2.16.840.1.114027.80.1.8'}]
   -
+    # OID scheme for composite variants:
```

Comment:
It's rather obvious given the reference to Entrust' OID space, but in line with #351 would you mind putting a comment at the top of the file referencing yourself (or anyone else taking that responsibility) for keeping these OIDs in line with underlying spec/draft (referencing the latter, too)?

Reply:
yes, of course.